src/AudioToolbox/AudioFormat.cs

```diff
@@ -165,15 +171,17 @@ Layout ToStruct ()
 #endif // !COREBUILD
 	}
 
+	[NoWatch]
 	public enum PanningMode : uint // UInt32 in AudioPanningInfo
 	{
 		SoundField					= 3,
 		VectorBasedPanning			= 4
 	}
 
+	[NoWatch]
 	public class AudioPanningInfo
 	{
-#if !COREBUILD
+#if !COREBUILD && !WATCH
```

Comment:
What exactly is COREBUILD?

Reply:
@tj-devel709 we need to build a _lightweight_ version of the platform assemblies to do bindings (e.g. so you can refer to the `AudioPanningInfo` inside ObjC bindings). However a lot of code would not compile (or would not be required). We use `COREBUILD` to controls what we need (include for bindings) and what's unneeded (not included).